### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by naftis** [uppi-fi/uppi.fi#49] [Ideas]
Maybe create a simple theming ability for the frontend and create some kind of dark theme?

**Comment by eioo** [comment]
Maybe we should ditch the current `theme.scss` file and start using something like [useTheme](https://usehooks.com/useTheme/) custom react hook to handle the theming?

There's also this but not too sure what it does and it has not been updated recently: https://github.com/donavon/use-dark-mode
